hack/testing-olm/test-040-eventrouter.sh

```diff
@@ -57,6 +57,25 @@ deploy_eventrouter() {
     fi
 }
 
+wait_elasticsearch() {
+  local looptries=${MAX_DEPLOY_WAIT_SECONDS}
```

Comment:
this is seconds but the looping is simply an ordinal number.  We should either:
* limit the number of tries to a max
* or keep track of time and continue until that time is exceeded

Reply:
i did it similar to the https://github.com/openshift/cluster-logging-operator/blob/master/hack/testing-olm/test-040-eventrouter.sh#L43